api/api.py

```diff
@@ -60,17 +62,18 @@ def get_filter_queries(earthquake_filters: dict[str]) -> list[str]:
     magtype = earthquake_filters[MAG_TYPE_FILTER_KEY]
     event = earthquake_filters[EVENT_FILTER_KEY]
     min_magnitude = earthquake_filters[MIN_MAGNITUDE_FILTER_KEY]
-    if status is not None:
+    logging.info(status)
+    if status not in FILTER_MISINPUTS:
         res.append(f"s.status = '{status}'")
-    if network is not None:
+    if network not in FILTER_MISINPUTS:
         res.append(f"n.network_name = '{network}'")
-    if alert is not None:
+    if alert not in FILTER_MISINPUTS:
         res.append(f"a.alert_value = '{alert}'")
-    if magtype is not None:
+    if magtype not in FILTER_MISINPUTS:
         res.append(f"mt.magtype_value = '{magtype}'")
-    if event is not None:
+    if event not in FILTER_MISINPUTS:
         res.append(f"t.type_value = '{event}'")
-    if min_magnitude is not None:
+    if min_magnitude not in FILTER_MISINPUTS:
         res.append(f"e.magnitude >= '{min_magnitude}'")
     if res:
         res[0] = "WHERE "+res[0]
```

Comment:
I believe this snippet of code would look better like this:
```
res = []
    if earthquake_filters[STATUS_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"s.status = '{earthquake_filters[STATUS_FILTER_KEY]}'")
    if earthquake_filters[NETWORK_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"n.network_name = '{earthquake_filters[NETWORK_FILTER_KEY]}'")
    if earthquake_filters[ALERT_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"a.alert_value = '{earthquake_filters[ALERT_FILTER_KEY]}'")
    if earthquake_filters[MAG_TYPE_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"mt.magtype_value = '{earthquake_filters[MAG_TYPE_FILTER_KEY]}'")
    if earthquake_filters[EVENT_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"t.type_value = '{earthquake_filters[EVENT_FILTER_KEY]}'")
    if earthquake_filters[MIN_MAGNITUDE_FILTER_KEY] not in FILTER_MISINPUTS:
        res.append(f"e.magnitude >= {earthquake_filters[MIN_MAGNITUDE_FILTER_KEY]}")
    if res:
        res[0] = "WHERE " + res[0]
    return res
```
what do you think?

Reply:
The reason I use variables here is to make it more legible. It gives each value an easier name to read, and makes the code less cluttered in my opinion.